src/main/java/com/alipay/oceanbase/rpc/protocol/payload/impl/ObTableSerialUtil.java

```diff
@@ -0,0 +1,131 @@
+package com.alipay.oceanbase.rpc.protocol.payload.impl;
+
+import com.alipay.oceanbase.rpc.ObGlobal;
+import com.alipay.oceanbase.rpc.protocol.payload.impl.execute.query.ObBorderFlag;
+import com.alipay.oceanbase.rpc.protocol.payload.impl.execute.query.ObNewRange;
+import com.alipay.oceanbase.rpc.util.Serialization;
+import io.netty.buffer.ByteBuf;
+
+import static com.alipay.oceanbase.rpc.protocol.payload.impl.ObTableObjType.*;
+
+public class ObTableSerialUtil {
```

Comment:
Maybe ObTableSerializeUtil is a better name>

Reply:
Because you use ObTableSerialUtil in server too, whether to change is depend on you

Reply:
maybe ObTableSerialUtil is too long, just consistent with sever